tesseract_core/runtime/experimental.py

```diff
@@ -30,6 +31,8 @@
 )
 from tesseract_core.runtime.schema_types import safe_issubclass
 
+IS_BUILDING = os.environ.get("_TESSERACT_IS_BUILDING", "0") == "1"
```

Comment:
Do we want to rename this, now that we're not just using it during build? Something like `SKIP_REQUIRED_FILE_CHECK`?